test.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+ROOT=$(cd "$(dirname ${BASH_SOURCE[0]:-$0})"; pwd)
+
+cd "$ROOT"
+go test $(find . -name '*_test.go' | sed 's/[^/]\+$//' | uniq)
```

Comment:
What advantage does this `$(find ...)` provide over `./...`

Reply:
I figured later on that `go test` can only run one package at the time, so I put it in a `for` loop, but I just expected to work if `*_test.go` files are popping up elsewhere. I was naive :smile: